source/scss/components/_linked-price.scss

```diff
@@ -37,7 +37,7 @@
       flex-grow: 1;
     }
 
-    div:last-child {
+    div:nth-last-child(1):not(.linked-price-value):not(.linked-price-info) {
       width: $linked-price-font-size;
     }
   }
```

Comment:
I know that this is not in the requirements, but I think it would be nice to have the `.linked-price-arrow` element display a `pointer` cursor on hover. It would make the Linked Price With Arrow element look different than the regular Linked Price element.

Reply:
Actually, the whole `.linked-price-body` element should have a pointer cursor on hover, but I just realised it has no difference form the regular Linked Price other than the `.linked-price-arrow` child element itself...

Reply:
You mean generally, we should add a pointer behaviour to the linked-price?

Reply:
I like the idea, but we shouldn't just do it. Should be clarified with Denise or Hanna

Reply:
Agree.